.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+
+go:
+  - 1.4.1
```

Comment:
We're using 1.4.2, rather than having to maintain this you should let make take care of pulling down the right version of go